docs/source/sctool/cluster.rst

```diff
@@ -80,32 +60,13 @@ Example (IPv6):
 
```

Comment:
@dgarcia360 TODO: Some YAML files have a property "example". I'll try to render the examples also from this new key.